### PR TITLE
feat: added grid component & updated links

### PR DESCRIPTION
**Pull request by zabihshaik**
Added grid component & updated links in other pages.

Grid page looks like:
Grid Page

Deploy preview link:
https://deploy-preview-11--neoz-ui.netlify.app/pages/grid.html

### CODE REVIEW

**Review by TejasShekar — APPROVED** [review]
Looks good to merge

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (TejasShekar) [inline .vscode/settings.json:2]: you can remove this file. not needed.

File: styles/grid.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    max-width: 100%;
+    padding: 1rem 0;
+    margin: 1rem auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    flex-wrap: wrap;    
+}
```
Comment (agrawalshanu708) [inline styles/grid.css:13]: use short methods in css

**Review by agrawalshanu708 — APPROVED** [review]
else looking good to merge

**Review by aiman-ansari — APPROVED** [review]
code is neat and clean everything looks good to merge

**Review by TejasShekar — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (zabihshaik) [reply .vscode/settings.json:2]: OK

**Pull request merged**